plugins/keysend.c

```diff
@@ -360,8 +361,14 @@ static struct command_result *htlc_accepted_call(struct command *cmd,
 		return htlc_accepted_continue(cmd, NULL);
 	}
 
+#if EXPERIMENTAL_FEATURES
+	allowed = cast_const(u64 *, FROMWIRE_TLV_ANY_TYPE);
+#else
 	/* We explicitly allow our type. */
-	allowed[0] = 5482373484;
+	allowed = tal_arr(cmd, u64, 1);
+	allowed[0] = PREIMAGE_TLV_TYPE;
+#endif
```

Comment:
Either way the `allowed` array will contain just one entry, and we'll check it below, so we could just add either `FROMWIRE_TLV_ANY_TYPE` or `PREIMAGE_TLV_TYPE` depending on whether we are `EXPERIMENTAL_FEATURES`, saving on allocations and `cast_const` trickeries.

Reply:
No, FROMWIRE_TLV_ANY_TYPE is literally a magic pointer... I will document this in a comment though.